service/rest_version.go

```diff
@@ -63,11 +64,12 @@ type restVersion struct {
 }
 
 type versionLessInfo struct {
-	Id       string         `json:"version_id"`
-	Author   string         `json:"author"`
-	Revision string         `json:"revision"`
-	Message  string         `json:"message"`
-	Builds   versionByBuild `json:"builds"`
+	Id       string              `json:"version_id"`
+	Author   string              `json:"author"`
+	Revision string              `json:"revision"`
+	Message  string              `json:"message"`
+	Builds   versionByBuild      `json:"builds"`
+	Tasks    versionStatusByTask `json:"tasks"`
```

Comment:
Does it make more sense to have tasks be a part of the build?

Reply:
I kind of figured it wouldn't matter but I guess it's not hard so, done.